lib/chain.ts

```diff
@@ -0,0 +1,29 @@
+
+export default class Chain {
+    /**
+     * roundAt determines the round number given a round time, a chain genesis and
+     * a chain period.
+     *
+     * @param time {number} Round time in ms
+     * @param genesis {number} Chain genesis time in ms
+     * @param period {number} Chain period in ms
+     */
+    static roundAt(time: number, genesis: number, period: number) {
+        if (time < genesis) return 1
+        return Math.floor((time - genesis) / period) + 1
+    }
+
+
+    /**
+     * roundTime determines the time a round should be available, given a chain
+     * genesis and a chain period.
+     *
+     * @param round {number} Round number
+     * @param genesis {number} Chain genesis time in ms
+     * @param period {number} Chain period in ms
```

Comment:
Genesis time is a Unix epoch in the Go version, so it is usually in seconds rather than ms.
Same for the period, the HTTP endpoints are currently exposing seconds.
Specify also maybe the return, I'm guessing it's `ms`.

Reply:
My point being: can't we have the `*1000` needed in the rest of the code being done here inside of that function instead, if JS is working in `ms`?

Reply:
Have updated this - it's technically a breaking change on the public API though, so have noted to make that clear in the release notes